frontend/src/routes/AccountPage.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component, PureComponent } from "react";
+import {Container} from 'flux/utils';
+import { dispatch } from '@Services/AppDispatcher.js';
+import PropTypes from 'prop-types';
+import { changePassword } from '@Actions/PasswordActions.js'
+import { Redirect } from 'react-router';
+import { FormGroup, Form,  FormControl, Alert, ControlLabel, Button, HelpBlock} from 'react-bootstrap';
+import {isEmptyString} from '@Utils/TypeChecks.js';
+import PasswordChangeComponent from '@Components/PasswordChangeComponent.jsx';
+
+//Data import
+import UserStore from '@Stores/UserStore.js';
+
+/**
+ * The login page. Since no state is needed, this is a Pure component that is rerendered
+ * only when new properties are provided.
+  */
+class AccountPage extends Component {
+    constructor(props) {
+        super(props);
+        
+        this.savePassword = this.savePassword.bind(this);
+    }
+    savePassword(id, pw){
+      //Apply the user based login action
+      dispatch(changePassword(this.props.user.id, this.state.newPassword));
```

Comment:
Dit moet geloof ik
```
dispatch(changePassword(this.props.user.id, pw));
```
zijn. Momenteel krijg ik errors omdat `this.state` undefined is.

Reply:
Scherp :-) Aangepast in nieuwe commit